deploy/1_Deployer.ts

```diff
@@ -35,6 +35,11 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
     expect(veErc721ExtensionAddress).to.be.equal(veErc721Extension.address);
     await accessControlSingleton.grantRole(await accessControlSingleton.ADMIN_ROLE(), mockErc721Extended.address);
 
+    const votingPower = await deployProxy("VotingPower", [accessControlSingleton.address, "StableUnit Voting Power", "vpSuDAO"]) as VotingPower;
+    let tx = await votingPower.setTokenWeight(veERC20.address,"1");
+    await tx.wait();
+    tx = await votingPower.setTokenWeight(veErc721Extension.address,"1");
+    await tx.wait();
     const mockErc721 = await deployProxy("MockErc721", ["Mock StableUnit NFT", "t_NFT"]);
```

Comment:
beware that we write mainnet deployer, and it just happens that we can use for testing. so IMHO deployer shouldn't have any mock logic whatsoever, and there's should be separte deploye script for this contacts, that checks that this is testnet/localhost

Reply:
I added TODO for this